tutorials/README.md

```diff
@@ -212,54 +217,53 @@ Videos can be restricted for playback in two very distinct ways. The first is ge
 
 ##### _Tutorial Sample:_
 
-> [Deleted Video](Tutorials)
+> [Deleted Video](/tutorials)
 
 #### Invalid or Unknown Videos
 
 Only valid, playable video IDs are accepted, and have their metadata pulled into the `Player` instance. If you end up with no `Player` instance, it’s because there are no valid video IDs for it to operate on. If you pass some ID that is incorrect, invalid, or unknown, the SDK has no idea what to do. So, you get an error to this effect. This tutorial sample shows how to detect that a video is not playable because the ID that was passed is invalid or unknown.
 
 ##### _Tutorial Sample:_
 
-> [Unknown Video](Tutorials)
+> [Unknown Video](/tutorials)
 
 ## Specific Notes for tvOS Apps
 
-The OVPSDK supports tvOS with the same source framework as iOS. We are utilising `AVPlayerViewController` for both content and advertisement playback. Controls also used from this class. As expected - no customisation of UI is allowed for tvOS implementation.
+The VVPSDK supports tvOS with the same source framework as iOS. We are utilising `AVPlayerViewController` for both content and advertisement playback. Controls also used from this class. As expected - no customisation of UI is allowed for tvOS implementation.
 
 Because there is no way to tap on the screen, you cannot access the ad URL. Additionally, tvOS has no support for web views – so there would be no consistent way to render the ad URL.
 
 <a name="tutorial-5"></a>
 ### Tutorial 5: Playing Videos on tvOS
 
-This tutorial sample shows you how to do many of the same things as iOS as described above in [Tutorial 1](#tutorial-1), but for tvOS. In terms of the OVPSDK, the biggest difference is that you cannot use the default Custom Controls UX with tvOS – you must use the built-in `AVPlayerViewController` controls. With this, you get direct access to the advanced Siri Remote control features, for example.
+This tutorial sample shows you how to do many of the same things as iOS as described above in [Tutorial 1](#tutorial-1), but for tvOS. In terms of the VVPSDK, the biggest difference is that you cannot use the default Custom Controls UX with tvOS – you must use the built-in `AVPlayerViewController` controls. With this, you get direct access to the advanced Siri Remote control features, for example.
 
 ##### _Tutorial Sample:_
 
-> [Playing Videos on tvOS](Tutorials/Tutorial_tvOS)
-
+> [Playing Videos on tvOS](/tutorials/Tutorial_tvOS)
```

Comment:
Should be `tutorials/sources/Tutorial_tvOS/`

Reply:
go ahead and fix the link

Reply:
please fix @RomanTysiachnik